ntbs-service/Services/NotificationService.cs

```diff
@@ -384,14 +384,15 @@ public async Task<Notification> CreateNewNotificationForUserAsync(ClaimsPrincipa
         {
             var defaultTbService = await _userService.GetDefaultTbService(user);
             var caseManagerEmail = await GetDefaultCaseManagerEmail(user, defaultTbService?.Code);
-
+            var caseManager = await _referenceDataRepository.GetUserByUsernameAsync(caseManagerEmail);
```

Comment:
I think it would be better to update the above method to `GetDefaultCaseManagerId` rather than doing this two step process.